Sming/SmingCore/HardwarePWM.cpp

```diff
@@ -75,39 +75,61 @@ bool HardwarePWM::analogWrite(uint8 pin, uint32 duty)
 	return setDuty(pin, duty);
 }
 
+/* Function Name: getDutyChan
+ * Description: This function is used to get the duty cycle number for a given channel
+ * Parameters: chan -Esp8266 channel number
+ */
+uint32 HardwarePWM::getDutyChan(uint8 chan)
+{
+	if(chan == PWM_BAD_CHANNEL)
```

Comment:
For consistency, we should be enclosing all conditional statements in braces { }